zubhub_frontend/zubhub/src/views/edit_profile/EditProfile.jsx

```diff
@@ -117,7 +117,7 @@ function EditProfile(props) {
                 className="auth-form"
                 name="signup"
                 noValidate="noValidate"
-                onSubmit={e => editProfile(e, props, toast)}
+                onSubmit={props.handleSubmit}
```

Comment:
this is how am handling data submission with the formik `handleSubmit`. check my comment here https://github.com/unstructuredstudio/zubhub/pull/1089/files#r1477202464

Reply:
see this https://github.com/unstructuredstudio/zubhub/pull/1089#discussion_r1477551368 for context on what I think about this whole thing.
This is the problem with making changes to code that is unrelated to your PR, you tend to inherit the regression introduced by the last person who touched that file, and now you'll need to make sure it works perfectly if the person doing the reviews notices the problems on time.